plugins/arDominionPlugin/css/less/bootstrap-mod.less

```diff
@@ -27,3 +27,8 @@
 .open > .dropdown-menu {
   z-index: 10004
 }
+
+// Upper padding for alerts
+.alert {
+  margin-top: 1em
```

Comment:
Good call... I'll make it more specific (and another change I should make is probably changing "#notice-alerts" to "#<notification type>-alert" so the ID only gets used once).